externals/sha3/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+project(SHA3)
+
+add_library(cvmfs_sha3 STATIC
```

Comment:
Is the 32bit/64bit header selection from `makeHook` taken care of?

Reply:
Good point, I forgot about 32bit. I will make an equivalent selection in the CMakeLists.txt.